postgresql.sql

```diff
@@ -0,0 +1,79 @@
+DROP TABLE IF EXISTS Stock;
```

Comment:
Did you execute this migration at first or did you automatically create the entities based on the TypeORM code?

Reply:
First, I generated the database, and then I realized that the database should be automatically generated by the entities of TypeORM, so I deleted it, at made it the right way: generated it with TypeORM